invenio_sitemap/views.py

```diff
@@ -0,0 +1,54 @@
+# -*- coding: utf-8 -*-
+#
+# Copyright (C) 2025 CERN.
+# Copyright (C) 2025 Northwestern University.
+#
+# invenio-sitemap is free software; you can redistribute it and/or modify
+# it under the terms of the MIT License; see LICENSE file for more details.
+
+"""Views."""
+
+
+from flask import Blueprint, abort, render_template
+from invenio_cache import current_cache
+
+from .cache import SitemapCache, SitemapIndexCache
+
+blueprint = Blueprint(
```

Comment:
it would be better to change to `create_blueprint(app)` approach [like](https://github.com/inveniosoftware/invenio-communities/blob/c03ec54c4472d8e06b53065e4c530afe8a3e2105/invenio_communities/views/ui.py#L129)

Reply:
I can see it's a necessity when the blueprint needs config data, but for now it doesn't, so I left it like this. Let's see if there is an appetite for configurability and change if desired.

Reply:
I agree. I don't see the need to make it a function for now. Typically we do that to allow configuring the endpoint, but in this case, `/sitemap` seems like the correct location for a general site map [source](https://www.sitemaps.org/protocol.html#location).

Reply:
Hearing from the zenodo-rdm folks would help for this, but until then as-is is fine.

Reply:
as i remember correctly, the change from a module variable to a function was made to avoid the problem with double instantiations error. so if that is not a problem with this package i am fine, although i like to have a consistent way of doing things